openedx/core/djangoapps/lang_pref/helpers.py

```diff
@@ -0,0 +1,36 @@
+"""
+Language preference cookie helper functions
+"""
+from django.conf import settings
+
+from . import COOKIE_DURATION
```

Comment:
nit: please use absolute path for the import.

Reply:
Done.